docs/integrations/sources/bing-ads.md

```diff
@@ -2,22 +2,44 @@
 
 This page contains the setup guide and reference information for the Bing Ads source connector.
 
+## Prerequisites
+- Microsoft Advertising account
+- Microsoft Developer Token
+- Reports start date
+
 ## Setup guide
 
-### Step 1: Set up Bing Ads
+<!-- env:oss -->
+
+For Airbyte Open Source set up your application to get **Client ID**, **Client Secret**, **Refresh Token** 
 
 1. [Register your application](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-register?view=bingads-13) in the Azure portal.
 2. [Request user consent](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-consent?view=bingads-13l) to get the authorization code.
 3. Use the authorization code to [get a refresh token](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-get-tokens?view=bingads-13).
 
-:::note
+4. :::note
 
 The refresh token expires in 90 days. Repeat the authorization process to get a new refresh token. The full authentication process described [here](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#access-token).
 Please be sure to authenticate with the email (personal or work) that you used to sign in to the Bing ads/Microsoft ads platform.
 :::
 
-4. Get your [Microsoft developer token](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token).
-5. If your OAuth app has a custom tenant and you cannot use Microsoft’s recommended common tenant, use the custom tenant in the **Tenant ID** field when you set up the connector.
+<!-- /env:oss -->
+
+### Step 1: Set up Bing Ads
+
+1. Get your [Microsoft developer token](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token).
+
+:::info
+
+To use Bing Ads APIs, you must have a developer token and valid user credentials.  You can follow these steps to get a developer token for production.
+    -  Sign in with [Super Admin](https://learn.microsoft.com/en-us/advertising/guides/account-hierarchy-permissions?view=bingads-13#user-roles-permissions) credentials at the [Microsoft Advertising Developer Portal](https://developers.ads.microsoft.com/Account) account tab. 
+    - Choose the user that you want associated with the developer token. Typically an application only needs one universal token regardless how many users will be supported. 
+    - Click on the Request Token button.
+See [Microsoft Advertising docs](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token) for more info.
+
+:::
```

Comment:
<img width="1136" alt="Screenshot 2023-11-03 at 18 41 05" src="https://github.com/airbytehq/airbyte/assets/5551758/3b3c02c5-f40f-4fb7-9a32-7879a0e5b896">
Do you know whye steps are rendered inside a code block?

Reply:
I agree, not the best way to show this steps, I added just list instead of this info block, thanks